lib/pages/ItemPage.dart

```diff
@@ -63,11 +64,11 @@ class ItemPageState extends State<ItemPage> {
     return Card(
       child: Column(
         children: <Widget>[
-          Text(item.name),
```

Comment:
Use table here. See Table on Customer.dart for reference

Reply:
alright looking into it

Reply:
added Table to ItemPage